versions.yaml

```diff
@@ -190,6 +190,11 @@ assets:
 externals:
   description: "Third-party projects used by the system"
 
+  attestation-agent:
+    description: "Provide attested key unwrapping for image decryption"
+    url: "https://github.com/confidential-containers/attestation-agent"
+    version: "v0.4.0"
+
```

Comment:
Why do we need the attestation-agent at this point?

Reply:
Removed.